templates/artist/artist_list.html

```diff
@@ -7,44 +7,34 @@
 {% endblock %}
 
 {% block content %}
-    <h1>Artists</h1>
+    <h2>Artists</h2>
     <div class="break">
         <hr>
     </div>
     <div class="list-nav">
         <div class="filter-sort-options center">
             <ul class="dropdown menu" data-dropdown-menu data-options="disableHover:true; clickOpen:true;">
                 <li class="nav-float"><h6>Show me</h6></li>
-                <li class="is-dropdown-submenu-parent opens-right nav-float">
-                    <a href="#"><h6>{{ campaign_status }}</h6></a>
-                    <ul class="menu is-dropdown-submenu">
-                        {% for option in campaign_statuses %}
-                            {% if option != campaign_status %}
-                                <li><a href="?campaign-status={{ option }}&genre={{ active_genre }}&sort={{ order_by.slug }}"><h6>{{ option }}</h6></a></li>
-                            {% endif %}
-                        {% endfor %}
-                    </ul>
-                </li>
-                <li class="nav-float"><h6>artists in</h6></li>
                 {% if genres %}
                     <li class="is-dropdown-submenu-parent opens-right nav-float">
-                        <a href="#"><h6>{{ active_genre }}</h6></a>
+                        <a href="#"><h6 class="list-font">{{ active_genre }}</h6></a>
                         <ul class="menu is-dropdown-submenu">
                             {% if active_genre != 'All Genres' %}
                                 <li><a href="?campaign-status={{ campaign_status }}&genre=All%20Genres&sort={{ order_by.slug }}"><h6>All Genres</h6></a></li>
                             {% endif %}
                             {% for genre in genres %}
                                 {% if genre != active_genre %}
-                                    <li><a href="?campaign-status={{ campaign_status }}&genre={{ genre|urlencode }}&sort={{ order_by.slug }}"><h6>{{ genre }}</h6></a></li>
+                                    <li><a href="?campaign-status={{ campaign_status }}&genre={{ genre|urlencode }}&sort={{ order_by.slug }}"><h6 class="list-font">{{ genre }}</h6></a></li>
                                 {% endif %}
                             {% endfor %}
                         </ul>
                     </li>
                 {% endif %}
+                <li class="nav-float"><h6>FROM</h6></li>
                 <li class="nav-float">
                     <button class="button hollow location" id="location-dropdown-button" type="button" data-toggle="location-dropdown">
                         <h6>{% if distance %}within {{ distance }} miles{% endif %}
-                        FROM {% if lat and lon %}my location{% elif location %}{{ location }}{% else %}ANYWHERE{% endif %}</h6>
+                        {% if lat and lon %}my location{% elif location %}{{ location }}{% else %}ANYWHERE{% endif %}</h6>
```

Comment:
Ahh.. I was trying to get the "from" to not be hyperlinked but didn't realize that it was connected to the other part. Changed it back